spec/classroom_spec.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'spec_helper'
+
+describe Classroom do
+  classroom = Classroom.new('104')
+  it 'should be an instance of Classroom' do
+    expect(classroom).to be_an_instance_of(Classroom)
+  end
+  it 'should have a label' do
+    expect(classroom.label).to eql('104')
+  end
```

Comment:
- [ ] Great job so far 👏🏻 👏🏻 However, according to the project requirements, there should be unit tests for all the methods in all the classes, Kindly make sure the `add_student` classroom method has (at least) an 'it' block 👇🏻 check your classroom.rb file

```ruby
  def add_student(student)
    @students << student
    student.classroom = self
  end
```